dashboard/test/lib/resources_autocomplete_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class ResourcesAutocompleteTest < ActiveSupport::TestCase
+  # We rely on fulltext indices to be used in this test.
+  # In order to get this test to work, we need to have the resources created
+  # as fixtures. These are defined in test/fixtures/resource.yml
```

Comment:
thanks for documenting. this strategy seems fine for now, though I'm a bit worried it will no longer work once we are only searching within a specific course version.